sources/gdal/large_image_source_gdal/__init__.py

```diff
@@ -140,6 +141,8 @@ def __init__(self, path, projection=None, unitsPerPixel=None, **kwargs):  # noqa
         self.tileSize = 256
         self.tileWidth = self.tileSize
         self.tileHeight = self.tileSize
+        if projection is None:
+            projection = config.getConfig('default_projection')
```

Comment:
We'll only want to do this IF the source is actually geospatial.  We currently have the isGeospatial method; this takes a path, so it should probably be broken into two methods where the current method calls a isDatasetGeospatial or the original function accepts a dataset or a path (and something has to be done equivalently in rasterio), and that if we are apply the default, we only do so IF the dataset is geospatial.

Reply:
Changed in [39e61be](https://github.com/girder/large_image/pull/1846/commits/39e61bea2aecab9268c68b17a1a7a470773c5b2c)